src/js/elements/subindicator_filter/filter_controller.js

```diff
@@ -197,17 +213,31 @@ export class FilterController extends Component {
         this.isLoading = false;
     }
 
-    addEmptyFilter(isDefault = false, isExtra = true, isRequired = false, addAsFirstRow = false) {
-        if (this.model.dataFilterModel.availableFilters.length > 0) {
-            const self = this;
+    addEmptyFilter(isDefault = false, isExtra = true, isRequired = false, addAsFirstRow = false, overrideLengthCheck = false) {
```

Comment:
I don't see `overrideLengthCheck` used anywhere, why would it be needed, are there plans to use it in future?